GDEdit Web/GDE.Web/Pages/FetchData.cshtml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Hopefully this is going to be removed, I don't wanna review every single mistake that doesn't follow the code style guidelines.

Reply:
this page will be removed once we get into making the other pages for the web app